zoo-css-challenge/style.css

```
@@ -64,6 +72,14 @@ button {
   background-color: #372772;
   border: 2px solid black;
   color: #fff;
+  font-size: 125%;
+}
+button:hover {
+  background-color:rgb(136, 136, 185);
+}
+
+button:focus {
+  background-color:rgb(136, 136, 185);
```

```suggestion
button:hover,
button:focus {
  background-color:rgb(136, 136, 185);
```
As these two selectors have the same CSS declaration, we can combine them like so